bin/zero-parachain-clean/node/Cargo.toml

```diff
@@ -0,0 +1,101 @@
+[package]
+name = "parachain-subzero-node"
+version = "0.1.0"
```

Comment:
```suggestion
version = "3.1.60"
```

Reply:
Or should it start from 0?

Reply:
Versioning approach is not clear to me for this runtime. Since it does not contain any Zero/GameDao features/pallets, is there any sense following same versioning now?
I was thinking of setting up correct version number at the moment, when we would enable these pallets in the runtime and leave it `0.1.0` for now, like a pre-release.
What's your opinion on that?

Reply:
- the versioning for runtime / node can be separate from zero pallets and gamedao pallets.
- also we need to come up with a release plan where we show node / runtime version x has zero / gamedao pallets y,z,a,b,... 

- the `version 1` as discussed for the stable env, how would that relate to the parachain versioning now? we should discuss maybe.